### DISCUSSION TITLE
Objective Value doesn't make sense...

### DISCUSSION THREAD

**Discussion by jakhon77** [google/or-tools#3610] [Routing (and legacy CP) questions] [answered]
In the documentation, the VRP example outputs the following. As it can be seen, the total miles traveled is 6208. However, solution.ObjectiveValue() shows 161408. Could someone explain to me what was calculated in the Objective Value? I thought the VRP tries to minimize the total miles traveled value, thus I was expecting the Objective Value to be equal to the Total Miles Traveled value. Thanks!

![image](https://user-images.githubusercontent.com/65193742/209826264-763518b6-0cf3-4c60-8d73-7065293ac546.png)

**Comment by Regista6** [comment, marked as answer]
https://groups.google.com/g/or-tools-discuss/c/YCUNTmWr8Us/m/f_hFmRP9BQAJ

Have a look at Mizux's answer.

In this case:
```
Distance of all routes + (max(all_route_end) - min(all_route_start)) * distance_dimension.GlobalSpanCostCoefficient ==
6208 + (max(1552,1552,1552,1552) - min(0,0,0,0)) * 100 ==
6208 + (1552-0) * 100 ==
6208 + 155 200 ==
161408
```

**Reply by jakhon77** [reply]
Understood! Thank you so much for the quick response!
